tests/unit/Owncloud/OcisPhpSdk/UserTest.php

```diff
@@ -113,6 +113,9 @@ public function testInvalidUserData(array $data)
             ]
         );
         $user = new SdkUser($libUser);
+        $user->getDisplayName();
+        $user->getMail();
+        $user->getId();
```

Comment:
the code will never get here, because `getDisplayName` throws the exception. If you want to test every function you will have to have more test cases